Boyardee updates 2

ToasterBan opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Another update to the Boyardee class 

- Meat fridge has extra meat
- Added a nutrivend
- Replaced the two empty monitors in the bridge with a civilian mod console and a power monitor
- Added a clothesmate
- Added a totally normal pet chicken and a totally normal strawbed for her
- Moved ice-cream vat to the cold room (why would you store your icecream right next to the engines?)
- PAI card moved from ontop of the helm to the bar
- Renamed external airlocks to make setting up the AAC easier
- AAC starts unlocked
- Added a sink inside of botany
- Added a completely normal chicken

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
I love but boyardee but it needs a little love, this update mostly makes it easier to produce food 
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Small tweaks around the Boyardee
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

ToasterBan committed:
Patch 1

ToasterBan committed:
Patch 2

ToasterBan committed:
fix

Apogee-dev commented:
Please update the Boyardee’s config file with ship name lists, outfits, and so on while you’re at it. Most of the older ships still don’t have those defined. Check one of the more up-to-date ships like the Carina if you need a reference.

ToasterBan committed:
config and waitors

triplezeta reviewed (approved):
ok!

MitztheKat commented:
fucking spess chicken

ToasterBan committed:
its a friday night

hey guys peter explaining the joke here

the ss13 server known as "shiptest" goes up wednesdays, fridays and saturdays and to people living  in GMT+1 timezones that is a night, in this commit a jukebox is added, with this and the commit name we realise that it is a reference to the "Yukaza 0" song known as "Friday Night"

Fikou commented on _maps/shuttles/shiptest/bar_ship.dmm:
```diff
@@ -1093,6 +1116,16 @@
+/obj/structure/bed/nest{
+	color = "#FFFF00";
+	desc = "It's a pile of soft, loose hay shaped like a nest.";
+	name = "Chicken Nest"
```
object names should be lower case

Fikou replied:
```suggestion
	name = "chicken nest"
```

Fikou committed:
Update _maps/shuttles/shiptest/bar_ship.dmm
